encoding/gocode/testdata/pointers.txtar

```diff
@@ -0,0 +1,57 @@
+-- in.cue --
+package generate
+
+import "github.com/grafana/thema"
+
+thema.#Lineage
+name: "pointers"
+seqs: [
+        {
+            schemas: [
+                {
+				    astring?: string
+					anint?:   int
+					abool?:   bool
+					alist?: [...string]
+					amap?: [string]: string
```

Comment:
let's:

* Separate this single case out into its own test
* Manually write the expected output (so the test will fail when run)
* Disable the new test using [the `ToDo` map](https://github.com/grafana/thema/blob/e665e4967517c5a2bbb7ac2f2f38d9a5f70441b1/internal/txtartest/txtar.go#L68)